src/Swashbuckle.AspNetCore.Filters/Examples/ExamplesOperationFilter.cs

```diff
@@ -53,17 +57,21 @@ private void SetResponseModelExamples(OpenApiOperation operation, OperationFilte
         {
             var responseAttributes = context.GetControllerAndActionAttributes<SwaggerResponseExampleAttribute>();
 
+            var mvcOptions = serviceProvider.GetService<IOptions<MvcOptions>>();
```

Comment:
Can all this be moved inside the ResponseExample class, instead of passing the outputFormatterSelector as a parameter?
Can we inject in an `IOptions<MvcOptions>` instead of asking the serviceProvider for one?

Reply:
Moved everything